manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Run administrative tasks."""
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "todo_list.settings")
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:
```

Comment:
The try-except block is used effectively to handle the potential ImportError. good practice